grading/load_config_json.cpp

```diff
@@ -20,6 +20,40 @@ void json_set_default(nlohmann::json &whole_config, std::string field, const T&
   }
 }
 
+uint64_t parse_file_size(std::string size_str) {
+  // Trim whitespaces
+  size_str.erase(std::remove_if(size_str.begin(), size_str.end(), isspace), size_str.end());
+
+  // Convert string to lowercase, note that it cannot handle UTF-8 characters
+  for (char& c : size_str)
+    c = std::tolower(c);
+
+  // Extract numeric file size, will cause exception if the number is larger than 64-bit
+  size_t unit_begin = 0;
+  uint64_t file_size = std::stoull(size_str, &unit_begin);
```

Comment:
If the `size_str` starts with a letter this line is going to give an error, which is not caught. (example: `k1234`)

Reply:
I think in this case, the program throws `invalid_argument()` from `std::stoull` as it could not be parsed from `pos = 0`...

Reply:
Yes the commad throws an error. Is it possible to use something like `regex_match` function in c++ to test the input value.  It would solve the below problem as well